src/Module.cpp

```diff
@@ -19,8 +19,7 @@ void Module::setDescription(const std::string &d) {
 
 AbstractCondition::AbstractCondition() :
 		makeRejectedInactive(true), makeAcceptedInactive(false), rejectFlagKey(
-				"Rejected") {
-
+				"Rejected"), rejectFlagValue( typeid(*this).name() ) {
```

Comment:
This would allow to check which module deactivated the candidate, right?

Reply:
Yes, this is the idea. This function was already implemented earlier but the value was not set correctly. I think the module name as default is a good starting option, without going through all modules and set it by hand.